solidity-v1/dashboard/src/pages/coverage-pools/CoveragePoolPage.jsx

```diff
@@ -168,8 +169,28 @@ const CoveragePoolPage = () => {
               )}
               amountClassName={"h3 text-grey-40"}
               symbolClassName={"h3 text-grey-40"}
+              wrapperClassName={"mb-2"}
             />
           </OnlyIf>
+          <span className={"text-grey-60 flex row"}>
+            <Icons.KeepSymbol
+              width={20}
+              height={20}
+              style={{ marginRight: "0.5rem", marginTop: "0.3rem" }}
```

Comment:
We're using `flex` in span wrapper so why do we need `marginTop`? can't we center elements in flex using `align-items: center` or `flex row center` to the `span` wrapper?

Reply:
We need this because we have multiline text next to the icon, so  simply centering the `span` wrapper unfortunately won't work. We want that icon to be centered with the first line only.